Game/menu.tscn

```diff
@@ -0,0 +1,60 @@
+[gd_scene load_steps=3 format=3 uid="uid://ctlv0f8nuv1wd"]
+
+[ext_resource type="Texture2D" uid="uid://s2g7jgj7r40t" path="res://boardwalkbetalogo.png" id="2_hr4hu"]
+
+[sub_resource type="GDScript" id="GDScript_piie1"]
+script/source = "extends Control
+
+
+func _on_start_pressed():
+	get_tree().change_scene_to_file(\"res://main.tscn\")
+
+
+func _on_exit_pressed():
+	get_tree().quit()
+"
```

Comment:
Would prefer this not being an inline script; this should be in a separate file such as `menu.gd`